Nos/Models/RelaySubscription.swift

```diff
@@ -42,7 +41,7 @@ struct RelaySubscription: Identifiable {
     ) {
         self.filter = filter
         self.relayAddress = relayAddress
-        self.id = (filter.id + "-" + relayAddress.absoluteString).sha256()
+        self.id = (filter.id + "-" + relayAddress.absoluteString).data(using: .utf8)!.sha256
```

Comment:
I know this could not fail in real life, but can you rewrite it without the `!`? We could just log that case so we can catch that unexpected case.

Reply:
I'm not sure why we are sha256ing this at all. It doesn't make it any more unique. I just removed that part.